processing/src/main/java/org/apache/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -478,6 +479,11 @@ public IncrementalIndexAddResult add(InputRow row) throws IndexSizeExceededExcep
     return add(row, false);
   }
 
+  public IncrementalIndexAddResult add(InputRow row, boolean skipMaxRowsInMemoryCheck) throws IndexSizeExceededException
+  {
+    return add(row, false, true);
```

Comment:
This appears to be ignoring `skipMaxRowsInMemoryCheck` is that intentional?  Probably worth a comment as to why that's intentional if it is.

Reply:
Thanks a lot for catching this! I must have missed it.